src/EnergyPlus/DaylightingManager.cc

```diff
@@ -9955,7 +9953,8 @@ void ReportIllumMap(EnergyPlusData &state, int const MapNum)
         state.dataDaylightingManager->ReportIllumMap_firstTime = false;
         FirstTimeMaps.dimension(state.dataDaylightingData->TotIllumMaps, true);
         EnvrnPrint.dimension(state.dataDaylightingData->TotIllumMaps, true);
-        RefPts.allocate(state.dataGlobal->NumOfZones, DataDaylighting::MaxRefPoints);
+        RefPts.allocate(state.dataGlobal->NumOfZones,
+                        state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints);
```

Comment:
Proper allocation :+1:

Reply:
Um, no, this isn't quite right. This allocation only happens once and will be based on the number of daylighting reference points in the zone of the first map. 

Looking at this more, `RefPts` appears to be used only in this function and only to build a title string for the map output. Why isn't this just a local string variable? And we really don't need to have `RefPts` and `refPt2`, just build the concatenated string directly from the base data in `ZoneDaylight`. I think `RefPts` can be removed from dataDaylightingData and just become a simple local string in this function.

Reply:
Hmmm...I missed that RefPts was only being used to pass the title field to the report.  So, from what I am seeing, it looks like it can be eliminated from the code and replace with a single character string that can be created and then passed downstream.  I'll try to get that fixed soon (today or tomorrow hopefully).

Reply:
@mjwitte Just committed a new version that has completely eliminated RefPts.  Hopefully ci comes back clean.